app/assets/src/components/views/SampleViewV2/ReportTable.jsx

```diff
@@ -326,13 +326,29 @@ class ReportTable extends React.Component {
       <div className={cs.stack}>
         <div
           className={cx(cs.stackElement, dbType == "nt" || cs.lowlightValue)}
-          onClick={onClick ? () => onClick[0]("nt") : null}
+          onClick={
+            onClick
+              ? () =>
+                  withAnalytics(
+                    onClick[0]("nt"),
+                    "ReportTable_count-type-nt_clicked"
+                  )
+              : null
+          }
         >
           {cellData ? cellData[0] : "-"}
         </div>
         <div
           className={cx(cs.stackElement, dbType == "nr" || cs.lowlightValue)}
-          onClick={onClick ? () => onClick[1]("nr") : null}
+          onClick={
+            onClick
+              ? () =>
+                  withAnalytics(
```

Comment:
The withAnalytics is supposed to receive a function. 
You can do 
```onClick={withAnalytics(
	() => onClick && onClick[1]("nr"),
	"ReportTable_count-type-nr_clicked"
```
Also, you have to add this to `renderNtNrSelector` function (not `renderNtNrValues`). I think the onClick here was a mistake.